pkg/kubectl/describe.go

```diff
@@ -269,7 +269,7 @@ func describePod(pod *api.Pod, rcs []api.ReplicationController, events *api.Even
 	return tabbedString(func(out io.Writer) error {
 		fmt.Fprintf(out, "Name:\t%s\n", pod.Name)
 		fmt.Fprintf(out, "Image(s):\t%s\n", makeImageList(&pod.Spec))
-		fmt.Fprintf(out, "Host:\t%s\n", pod.Spec.Host+"/"+pod.Status.HostIP)
+		fmt.Fprintf(out, "Host:\t%s\n", pod.Spec.NodeName+"/"+pod.Status.HostIP)
```

Comment:
I will update pther places with similar problem. I was not sure which way was better so I didn't update them.